tree_sitter_grep_plugin_core/src/lib.rs

```diff
@@ -0,0 +1,33 @@
+use tree_sitter::Node;
+
+// per https://adventures.michaelfbryan.com/posts/plugins-in-rust/
+
+pub trait Filterer: Send + Sync {
```

Comment:
It seemed like I needed to add this `Send + Sync` bound in order to get the (`static`) `OnceCell` to be happy?